src/content/docs/errors-inbox/browser-tab.mdx

```diff
@@ -0,0 +1,147 @@
+---
+title: 'Browser: Group errors tab '
+tags:
+  - APM
+  - APM UI pages
+  - Error analytics
+metaDescription: New Relic's browser group errors tab shows trends and anomalies that help you compare and troubleshoot error events.
+---
+import errorsinboxBrowserGroupErrorsTab from 'images/errors-inbox_screenshot-full_browser-group-errors-tab.webp'
+
+import errorsinboxBrowserErrorsListView from 'images/errors-inbox_screenshot-full_browser-errors-list-view.webp'
+
+import errorsinboxBrowserErrorProfiles from 'images/errors-inbox_screenshot-crop_browser-error-profiles.webp'
+
+import errorsinboxBrowserErrorsDetailsView from 'images/errors-inbox_screenshot-crop_browser-errors-details-view.webp'
+
+import errorsinboxErrorsInboxTriageSection from 'images/errors-inbox_screenshot-crop_errors-inbox-triage-section.webp'
+
+import errorsinboxEventTrailforErrorsInbox from 'images/errors-inbox_screenshot-crop_event-trail-for-errors-inbox.webp'
+
+import errorsinboxErrorsInboxStackTrace from 'images/errors-inbox_screenshot-crop_errors-inbox-stack-trace.webp'
+
+With **Group errors** tab you can dynamically filter and group errors for deeper analysis.
```

Comment:
I'd add more content in the intro. Maybe you can add here what you have below the screenshot:
Group errors shows you where your JavaScript errors are happening, and provides tools to help you figure out the root cause. You can also query and create dashboards of JS error data in the [query builder](/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder), or use the browser API to monitor handled errors.